CHANGELOG.md

```diff
@@ -28,6 +28,81 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v7.3.0](https://github.com/shopsys/shopsys/compare/v7.2.2...v7.3.0) - 2019-07-04
```

Comment:
Uf, I had a quite hard time to review the changelog :slightly_frowning_face: 
Thanks for providing me the original dump and information what you have added manually :+1:  - I have some thoughts here :smile: 
- I do agree with not including changes that just fixed something on the master (BC compatibility improvements like #1131, #1143, etc.) so they are not relevant in the context of viewing the changes since the previous release
- I rather agree with not including typo-like fixes (#1160, #1080, ...), however, I do not like the fact that we need to think about it...
- I disagree with not including #1158, #1172, #1162, and I am not sure about the other not included changes - #1130, #1179, #1128...

So, to sum it up, I think you should include #1158, #1172, #1162 into the changelog and I think we really need to do something with the automatic dumping to make it work with multiple branches better...there is too much manual work in this step at the moment :disappointed:

Reply:
About #1158 – It's not included intentionally because the problem was solved more systematically in #1161 and the solution from 1158 is not in the codebase anymore.
#1172 – okay, added
#1162 – hmm... okay, this information may be beneficial to the users. Added.

And I don't think the fixes and improvements in documentation should be a part of changelog (especially when it's only one sentence) – #1130, #1179, #1128

Reply:
Thanks for the answer and fixes :ok_hand: